src/app/model.ts

```diff
@@ -44,9 +44,9 @@ export type Connections = Array < Connection > ;
 
 export interface Connector extends BaseEntity {
     icon: string;
+    description: string;
```

Comment:
Ah, no worries.. just OCD tendencies.. btw just waiting on CircleCI to finish up.